### PR TITLE
Add revision model unit test, add null check

### PR DESCRIPTION
**Pull request by odockal**
Signed-off-by: Ondrej Dockal <odockal@redhat.com>

### CODE REVIEW

File: test/knative/revision.test.ts [hunk 1]
```diff
@@ -0,0 +1,109 @@
+import { expect } from 'chai';
+import * as revision from '../../src/knative/revision';
+
+suite('Revision class', () => {
+  let instance: revision.Revision;
+  const annotationObj: any = {};
```
Comment (joshuawilson) [inline test/knative/revision.test.ts:6]: Please update from `any` to the type in the `revision.ts`
